dimod/binary_quadratic_model.py

```diff
@@ -157,9 +190,10 @@ def __init__(self, linear, quadratic, offset, vartype, **kwargs):
 
     @classmethod
     def empty(cls, vartype):
-        """Create an empty BinaryQuadraticModel.
+        """Create an empty :class:`.BinaryQuadraticModel` instance.
```

Comment:
I am not sure this needs to be a hyperlink everywhere. Especially when it is a method of the mentioned class. The first line of a docstring is a bit special in that IDEs will use it to prompt the user when autocompleting or whatever. So I would avoid sphinx-specific syntax in the header line

Reply:
Fixed